tests/test_tree_tests.sh

```diff
@@ -5,24 +5,36 @@ function mtime() {
 }
 
 # Go to top of blueprint tree
-TOP=$(dirname ${BASH_SOURCE[0]})/..
-cd ${TOP}
+cd $(dirname ${BASH_SOURCE[0]})/..
+TOP=${PWD}
 
-rm -rf out.test
-mkdir out.test
+export TEST_OUT=/tmp/blueprint.out.test
+rm -rf "${TEST_OUT}"
+mkdir "${TEST_OUT}"
 
-rm -rf src.test
-mkdir src.test
-cp -r tests/test_tree src.test/test_tree
-ln -s ../.. src.test/test_tree/blueprint
 
-cd out.test
-export SRCDIR=../src.test/test_tree
+export TEST_SRC_LINK=/tmp/blueprint.src.test
+rm -rf "${TEST_SRC_LINK}"
+mkdir "${TEST_SRC_LINK}"
+cp -r tests/test_tree "${TEST_SRC_LINK}"/test_tree
+ln -s "${TOP}" "${TEST_SRC_LINK}"/test_tree/blueprint
+
+cd "${TEST_OUT}"
+
+export SRCDIR="${TEST_SRC_LINK}/test_tree"
```

Comment:
This is needed for bootstrap.bash but not cleanup, move it just above bootstrap.bash?